test/services/LeafValueCoder.test.ts

```diff
@@ -24,11 +25,9 @@ describe('LeafValueCoder', () => {
         [1.900719925474099, '1a60b6b3a763ff38'],
       ];
 
-      dataToTest.forEach((arr) => {
-        const [toConvert, expected] = arr;
-
+      dataToTest.forEach(([toConvert, expected]) => {
         it(`expect to convert ${toConvert} to solidity uint256`, () => {
-          const encoded = LeafValueCoder.encode(toConvert).toString('hex');
+          const encoded = LeafValueCoder.encode(toConvert, '').toString('hex');
```

Comment:
what `''` means as the second parameter?

Reply:
empty/any label